apps/postybirb-ui/src/components/translations/validation-translation.tsx

```diff
@@ -68,28 +68,28 @@ export const TranslationMessages: Partial<TranslationsMap> = {
     const { maxLength, currentLength } = props.values;
     return (
       <Trans>
-        Tag limit reached ({currentLength} / {maxLength})
+        Tag limit reached ({currentLength}/{maxLength})
```

Comment:
I find this less readable imo

Reply:
hmmmm

Tag limit reached (7/5)
Tag limit reached (7hmmm / 5)

i see now, will revert. Possibly will add this as a comment